etc/kayobe/ansible/pci-passthrough.yml

```diff
@@ -0,0 +1,143 @@
+---
+- name: Enable GPU passthough
+  hosts: "{{ (gpu_group_map | default({})).keys() }}"
+  vars:
+    # This playbook will execute after nodes are deployed
+    # and before overcloud host configure - we can't assume
+    # users and venvs exist.
+    ansible_user: "{{ bootstrap_user }}"
+    ansible_ssh_common_args: "-o StrictHostKeyChecking=no"
+    ansible_python_interpreter: "/usr/bin/python3"
+    vfio_pci_ids: |-
+      {% set gpu_list = [] %}
+      {% set output = [] %}
+      {% for gpu_type in gpu_group_map | dict2items | default([]) %}
```

Comment:
todo: change `gpu_type` to something like `gpu_group`, since it's looping on group names to gpu types

Reply:
Done